src/ops/ee/composition_config_generator.py

```diff
@@ -0,0 +1,113 @@
+# Copyright 2019 Adobe. All rights reserved.
```

Comment:
@costimuraru WOuld it make sense to move `src/ops/ee` to `src/ops/hierarchical` package ?

Reply:
Done